src/main/java/athena/task/taskfilter/FlexibleTimeFilter.java

```diff
@@ -20,12 +20,8 @@ public FlexibleTimeFilter(Boolean isFlexible) {
      */
     @Override
     public boolean isTaskIncluded(Task task) {
-        if (task.getDates() == null) {
-            return isFlexible;
-        }
-        return !isFlexible;
+        return task.isFlexible();
```

Comment:
The `isFlexible` attribute of this class is to determine whether to get flexible tasks or not flexible tasks. So, this needs to be `task.isFlexible() ^ isFlexible`. But anyways I think `isFlexible` is not very clear. How about something like `filterFlexibleTask` or `doesIncludeFlexibleTask`